vendor/views/vendor_admin.py

```diff
@@ -331,3 +330,20 @@ def get(self, request, *args, **kwargs):
 
         messages.info(request, _("Offer Added To Customer Profile"))
         return redirect(reverse('vendor_admin:manager-profile', kwargs={'uuid': customer_profile.uuid}))
+
+
+class ProductAvailabilityToggleView(LoginRequiredMixin, View):
+    success_url = reverse_lazy('vendor_admin:manager-product-list')
+
+    def post(self, request, *args, **kwargs):
```

Comment:
Definitely looks like this'll work but wonder why we aren't just using an UpdateView?

Reply:
Don't know did not really think about it. Was treating it as a simple endpoint not like a View